src/main/java/seedu/address/model/folder/UniqueFolderList.java

```diff
@@ -48,6 +49,21 @@ public void add(Folder toAdd) {
         internalList.add(toAdd);
     }
 
+    /**
+     * Adds a contact to the folder.
+     * The contact must not already exist in the folder.
+     */
+    public void addContact(Person contact, FolderName name) {
+        requireNonNull(contact);
+        for (int i = 0; i < internalList.size(); i++) {
+            Folder folder = internalList.get(i);
+            if (folder.getFolderName() == name) {
+                folder.addContacts(contact);
+            }
+        }
```

Comment:
Since every folder name is unique, is it possible to break out of the `for` loop once a folder name matches and a contact is added to the matched folder to make it slightly more efficient?

Can also consider using streams to do the filter. 
```
internalList.stream().filter(folder -> folder.getFolderName() == name).findFirst().get().addContacts(contact)
```
I did not test the code above, but you can try if you want to!